scripts/cluster-upgrade.sh

```diff
@@ -226,24 +232,25 @@ upgrade() {
     
     log $OCM_NAME "upgrade" "Checking $OCM_NAME..."
 
-    # - do we need to upgrade?
-    OCP_CURRENT_VERSION=$(oc -n $CD_NAMESPACE get clusterdeployment $CD_NAME -o json | jq -r '.status.clusterVersionStatus.history[] | select(.state == "Completed") | .version' | head -n1)
+    # do we need to upgrade?
+    OCP_CURRENT_VERSION=$(KUBECONFIG=$TMP_DIR/kubeconfig-${CD_NAMESPACE} oc get clusterversion version -o json | jq -r '.status.history[] | select(.state == "Completed") | .version' | grep $FROM)
+    KUBELET_VERSION_COUNT=$(KUBECONFIG=$TMP_DIR/kubeconfig-${CD_NAMESPACE} oc get nodes --no-headers -o custom-columns=VERSION:.status.nodeInfo.kubeletVersion | sort | uniq | wc -l)
```

Comment:
```suggestion
    KUBELET_VERSION_COUNT=$(KUBECONFIG=$TMP_DIR/kubeconfig-${CD_NAMESPACE} oc get nodes --no-headers -o custom-columns=VERSION:.status.nodeInfo.kubeletVersion | sort -u | wc -l)
```

What's the count of the unique kubelet versions do for us?

Also, `sort -u` is one fewer pipe for `sort | uniq`

Reply:
If we have more than one version on our nodes the worker upgrades are not done.  Thanks for the suggestion.. `sort | uniq` is muscle memory I should change :(